### PR TITLE
[Internal] Patch: Fixes JSON Patch Syntax

### PR DESCRIPTION
**Pull request by deparash**
Minor update at backend

# Pull Request Template

## Description

a minor update at backend is done that needs a update on SDK side.
Basically inc for increment is now changed to incr

## Type of change

Please delete options that are not relevant.

- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

**Review by abhijitpai — APPROVED** [review]

**Comment by rakkuma** [comment]
Thanks Devansh.

**Review by rakkuma — APPROVED** [review]

File: Microsoft.Azure.Cosmos/src/Patch/PatchConstants.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public static class OperationTypeNames
             public const string Remove = "remove";
             public const string Replace = "replace";
             public const string Set = "set";
-            public const string Increment = "inc";
+            public const string Increment = "incr";
```
Comment (j82w) [inline Microsoft.Azure.Cosmos/src/Patch/PatchConstants.cs:24]: Is there no test validating this?

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Review by j82w — APPROVED** [review]
Please create a backlog item to add an end to end test for all the operations.

File: Microsoft.Azure.Cosmos/src/Patch/PatchConstants.cs (hunk 1 above)
Comment (deparash) [reply Microsoft.Azure.Cosmos/src/Patch/PatchConstants.cs:24]: yes backend code was not updated. I've created a backlog item for this

**Pull request merged**